src/components/Table/components/TableCell/TableCell.tsx

```diff
@@ -24,10 +27,23 @@ const TableCell: React.FC<Props> = React.memo(
     children,
     type = 'normal',
     padded = false,
+    dataTestIdPrefix,
+    rowIndex,
```

Comment:
Not sure if a problem, but since both rowIndex and index can be undefined, this can lead to testIds such as 'table_row_undefined_cell_undefined'
<img width="454" alt="table-pr" src="https://user-images.githubusercontent.com/24905735/118144846-9a3e0200-b415-11eb-8226-21b4b15d0f74.png">

Reply:
resolved